### PR TITLE
Create stale action

### PR DESCRIPTION
**Pull request by jhnlsn**
# Description of the issue
Creating stale action config

# Description of changes
This will mark issues stale after a period of inactivity

# License
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

# Tests
_Describe what tests you have done._

### CODE REVIEW

**Comment by SaxyPandaBear** [comment]
I think the YAML file looks fine, but from https://github.com/actions/stale (got here from [this](https://github.com/marketplace/actions/stale-for-actions deprecated GitHub action that uses a `.github/stale.yml` file) I don't see how this YAML fits. It looks like from their documentation, the action is defined in a workflow now.

**Comment by jhnlsn** [comment]
Oh good catch! When I saw the yaml based config, I didn't see it as deprecated.  Must have missed it.

**Comment by jhnlsn** [comment]
This is what I had gone off of before. https://github.com/marketplace/stale

**Review by SaxyPandaBear — DISMISSED** [review]
LGTM

File: .github/workflows/close-stalled.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: Close stalled issues and PRs
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+permissions:
+  issues: write
```
Comment (SaxyPandaBear) [inline .github/workflows/close-stalled.yml:7]: Ah, missed this from the last review. My bad.

File: .github/workflows/close-stalled.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+          days-before-issue-stale: 90
+          stale-issue-message: 'This issue was marked stale due to lack of activity.'
+
+          days-before-issue-close: 30
+          close-issue-message: Closing this because it has stalled. Feel free to reopen if this issue is still relevant, or to ping the collaborator who labeled it stalled if you have any questions.
+
+          # Do not close PRs
+          days-before-pr-close: -1
+          # close-pr-message: Closing this because it has stalled. Feel free to reopen if this PR is still relevant, or to ping the collaborator who labelled it stalled if you have any questions.
```
Comment (SaxyPandaBear) [inline .github/workflows/close-stalled.yml:30]: Honestly, I feel like we should just remove this instead of leaving it in commented.

**Pull request merged**